### DISCUSSION TITLE
How to increase kafka consumption speed

### DISCUSSION THREAD

**Discussion by 1450793561** [apache/skywalking#11919] [General]
skywalking9.7.0 through the kafka channel, queue backlog is serious, how to improve the consumption speed?
<img width="881" alt="企业微信截图_07d9c275-0683-4e1e-9a72-ced6092fa3c5" src="https://github.com/apache/skywalking/assets/31468018/e432e98a-bc91-42f7-a1bd-9d5db952ed8e">

**Comment by wu-sheng** [comment]
Duplicate https://github.com/apache/skywalking/discussions/11928
